### ISSUE TITLE
File attachement / samples size increase explanation

### ISSUE THREAD

**Issue by philpraxis** [MISP/MISP-sizer#4]
Add text:
"Add the space to Disk size that you estimate will be taken by your uploaded samples. This is not integrated in the calculation, as any file attachement is up to you and optional".

**Comment by iglocska** [comment]
Good point indeed. Maybe something to note is that the file samples can be stored externally too, but additional configuration is needed if you go down that route.

**Comment by philpraxis** [comment]
Yes ideally we should add link from this sizer page to relevant help pages as this sizing question is probably one of the first question a serious newcomer will investigate.
So if we have links here, it makes a gentle gradual engagement which will increase adoption and smooth learning curve.

**Comment by iglocska** [comment]
First we need the relevant help pages though *cough* ;)
